borsh/src/schema_helpers.rs

```diff
@@ -19,6 +26,18 @@ pub fn try_from_slice_with_schema<T: BorshDeserialize + BorshSchema>(v: &[u8]) -
     Ok(object)
 }
 
+pub fn from_slice<T: BorshDeserialize>(v: &[u8]) -> Result<T> {
```

Comment:
* [ ] Let's move this function to `src/de/mod.rs`, as it is more appropriate place for it
* [ ] Add documentation with a usage example, so users can easily copy-paste the code and become productive faster (take inspiration from [`serde_json`](https://docs.rs/serde_json/latest/serde_json/fn.from_slice.html))